store/src/file_upload.rs

```diff
@@ -38,17 +34,12 @@ impl FileUpload {
     pub async fn run(self, shutdown: &triggered::Listener) -> Result {
         tracing::info!("starting file uploader");
 
-        let enabled = self.enabled;
```

Comment:
some sort of testing gate i assume

Reply:
yeah, never really used it and just use an invalid bucket name to avoid uploads